### PR TITLE
fix(k8s_loader): make docker base loader work

### PR DESCRIPTION
**Pull request by fruch**
60735f3fc4f5d0b70ec7bd616341bb9826d4fa44 splited the loader configuration into 2 files, but dropped the docker mount which is still needed, until we refactor the code to use the dynamic loaders approch.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
testing in:
https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fruch/job/longevity-scylla-operator-3h-eks/20/

**Comment by vponomaryov** [comment]
What do you need to be as part of the mentioned `refactor`?
`Dynamic` approach is already suitable for usage by any K8S job because we do not use `profiles` (copying files) there.
If we start using profiles, then it is not refactor, but improvement by adding 1 small feature to support `send_files` interface.

Then, about the `static` approach. Where do you need the docker socket?

**Comment by fruch** [comment]
> What do you need to be as part of the mentioned `refactor`? `Dynamic` approach is already suitable for usage by any K8S job because we do not use `profiles` (copying files) there. If we start using profiles, then it is not refactor, but improvement by adding 1 small feature to support `send_files` interface.
> 
We need refactoring of the stress threads to be using it, right now all of them are calling DockerRemote, also we'll need to add an option to select the image, since each is using a different one 

> Then, about the `static` approach. Where do you need the docker socket?
currently in the static, we still need a way for current DockerRemote code to work, i.e. scylla-bench inside nemesis, is currently failing, since you removed the docker socket

This is a patch to get things working again, as they were. we still need some amount of recatoring working to align all the stress thread code to be using the `dynamic` approach

File: sdcm/k8s_configs/loaders/sts.yaml [hunk 1]
```diff
@@ -31,4 +31,20 @@ spec:
             requests:
               cpu: ${POD_CPU_LIMIT}
               memory: ${POD_MEMORY_LIMIT}
+          securityContext:
```
Comment (vponomaryov) [inline sdcm/k8s_configs/loaders/sts.yaml:34]: Please, add a "TODO" here stating that we should remove it right after the stress logic is handled by K8S on K8S backend and not bare docker.

**Review by vponomaryov — CHANGES_REQUESTED** [review]

**Review by vponomaryov — APPROVED** [review]

**Pull request merged**